Dockerfile

```diff
@@ -6,11 +8,7 @@ RUN curl -sL https://deb.nodesource.com/setup_${NODE_VERSION}.x | bash -
 
 RUN apt-get update && \
   apt-get install -y \
-    nodejs
-
-RUN pip install pipenv
+    nodejs && \
+  pip install pipenv
```

Comment:
This reduces the number of layers in our Docker image, which helps with performance (also, Heroku doesn't run containers with over 40 layers).